docs/data-sources/servicestage_component_runtimes.md

```diff
@@ -0,0 +1,38 @@
+---
+subcategory: "ServiceStage"
+---
+
+# huaweicloud_servicestage_component_runtimes
+
+Use this data source to query available runtimes within HuaweiCloud.
+
+## Example Usage
+
+```hcl
+data "huaweicloud_servicestage_component_runtimes" "test" {}
+```
+
+## Argument Reference
+
+* `region` - (Optional, String) Specified the region in which to obtain the component runtimes.
+  If omitted, the provider-level region will be used.
+
+* `name` - (Optional, String) Specified the runtime name to use for filtering.
+
+* `default_port` - (Optional, Int) Specified the default container port to use for filtering.
+
+## Attributes Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - The data source ID.
+
+* `runtimes` - The list of runtime details.
+
+The `runtimes` block contains:
+
+* `name` - The runtime name.
+
+* `default_port` - The default container port.
+
+* `description` - The type description.
```

Comment:
s/The type description./The runtime description.

Reply:
the description has been updated.